src/Renderer/Mesh/Mesh.h

```diff
@@ -40,6 +40,7 @@ namespace SnekVk
         public:
 
         static constexpr size_t MAX_VERTICES = 10000;
+        static constexpr size_t MAX_INDICES = 100000;
```

Comment:
where do these values come from?

Reply:
I made them up. They're arbitrary rn, we can work out the full requirements for these later on.